src/registry_schemas/example_data/schema_data.py

```diff
@@ -50,7 +50,7 @@
     'lastLedgerTimestamp': '2019-04-15T20:05:49.068272+00:00',
     'lastPreBobFilingTimestamp': '2019-04-15T20:05:49.068272+00:00',
     'legalName': 'legal name - CP1234567',
-    'businessName': 'legal name - CP1234567',
+    'alternateNames': 'legal name - CP1234567',
```

Comment:
Not sure if I should carry on editing this file, if yes I will continue, if no I will revert these changes

Reply:
You can replace both instances of `businessName` with `"alternateNames": []`.

This is what the legal api currently returns when there are no alternate name entries.